packages/rum-react/test/e2e/react/webpack.config.js

```diff
@@ -31,19 +31,39 @@ module.exports = {
   entry: path.resolve(__dirname, './app.jsx'),
   output: { path: __dirname, filename: 'app.e2e-bundle.js' },
   devtool: 'source-map',
-  mode: 'production',
+  mode: 'development',
   performance: {
     hints: false
   },
   module: {
     rules: [
       {
         test: /.jsx?$/,
+        exclude: /node_modules/,
         use: {
-          loader: 'babel-loader'
+          loader: 'babel-loader',
+          options: {
```

Comment:
not required to have this config again here.

Reply:
I've tested it and it doesn't work without the options.

Reply:
https://github.com/elastic/apm-agent-rum-js/issues/308

Reply:
Yeah sounds good,  But lets do this suggestion for now.  https://github.com/elastic/apm-agent-rum-js/pull/265/files#r300721927